pyproject.toml

```diff
@@ -51,6 +52,7 @@ rich = "13.*"
 cleo = "^2.1.0"
 hatchling = "^1.21.0"
 typer = "0.*"
+pyyaml = "^6.0.2"
```

Comment:
Maybe it is difficult without pyyaml, but would it be possible to parse the YAML file without it? Otherwise, I think the versioning should be "*" to not collide with projects using Polylith and also using PyYaml in their project.

Reply:
Well, I might parse the lockfile without pyyaml, but it makes it straigthforward to extract the pypi package names and versions. Plus, the `safe_load` method prevents from executing insecure, arbitrary code embedded in a yaml file.

I will correct the versioning, though

Reply:
Done in 1e7f20d

Reply:
Thank you!